plugins/aws/mq/mqBrokerPublicAccess.js

```diff
@@ -0,0 +1,66 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'MQ Broker Public Accessibility',
+    category: 'MQ',
+    domain: 'Application Integration',
+    description: 'Ensure that Amazon MQ brokers are not publicly accessible from the internet.',
+    more_info: 'Public Amazon MQ brokers can be accessed directly, outside of a Virtual Private Cloud (VPC), therefore every machine on the Internet can reach your brokers through their public endpoints and this can increase the opportunity for malicious activity.',
```

Comment:
```suggestion
    more_info: 'Public Amazon MQ brokers can be accessed directly, outside of a Virtual Private Cloud (VPC), therefore every machine on the internet can reach your brokers through their public endpoints and this can increase the opportunity for malicious activity.',
```